datasets/us/doh_debarred/us_doh_debarred.yml

```
@@ -0,0 +1,57 @@
+name: us_doh_debarred
+title:  US DOH Inspector General Exclusions
+entry_point: crawler.py
+coverage:
+  frequency: daily
+  start: "2024-03-13"
+summary: >
+  Peoples and companies excluded from Federally funded health care programs
+description: |
+  This dataset provides information on Individuals and entities excluded from participation in Medicare,
+  Medicaid, and all other Federal health care programs for various reasons including conviction for Medicare 
+  or Medicaid fraud.
+  Exclusions are imposed under the authority of sections 1128 and 1156 of the Social Security Act.
+  
+  From the OIG HHS page:
```

Isn't it the other way around?

```suggestion
  From the HSS OIG page:
```